Source/XCDBLD/FrameworkBundle.swift

```diff
@@ -11,25 +11,24 @@ import Result
 /// - parameter platformName: If given, only sends bundles from an XCFramework with a matching `SupportedPlatform`.
 /// - parameter variant: If given along with `platformName`, only sends bundles from an XCFramework with a matching `SupportedPlatformVariant`.
 public func frameworkBundlesInURL(_ url: URL, compatibleWith platformName: String? = nil, variant: String? = nil) -> SignalProducer<Bundle, DecodingError> {
-	guard let bundle = Bundle(url: url) else {
-		return .empty
-	}
-
-	switch bundle.object(forInfoDictionaryKey: "CFBundlePackageType") as? String {
-	case "XFWK":
+	switch url.pathExtension {
+	case "xcframework":
```

Comment:
Checking for an "xcframework" path extension is new behavior, but I think this is safe. `xcodebuild -create-xcframework` forces you to output to a bundle directory with an `.xcframework` extension, and Xcode only extracts an xcframework at build-time if it ends in `.xcframework`. 

So unless there are some intermediate-and-badly-named XCFrameworks passing around inside Carthage, this should match them all.